README.md

```diff
@@ -1 +1,47 @@
-# Github Action Sync Branches
\ No newline at end of file
+# Github Action Sync Branches
+
+An action to create a pull request between two branches.
+
+## Example Usage
+
+This action can be used to create a sync pull request when a branch gets a closed PR. This helps to keep downstream branches in sync.
+
+Here's an example of a `.github/workflows/*.yml` file to trigger the action:
+
+```
+name: Sync master/develop branches
+
+on:
+  pull_request:
+    types: [closed]
+    branches: master
+
+jobs:
+  create_sync_pull_request:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dequelabs/action-sync-branches@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-title: "chore: merge master into develop"
+          pr-reviewers: scurker
+          pr-labels: chore
+```
+
+## Configuration
+
+- **github-token** (required) - needed to interact with github api
```

Comment:
This is provided by default, right? Or do we need to generate a special token just for this tool?

Reply:
Yup, github provides a [short lived temporary access token](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token) only for this repo.

Reply:
Can we mention this in the config documentation then? The way this reads suggests to me that I need to _provide_ a `github-token` value.

Reply:
You still have to pass it into the action:

> To use the `GITHUB_TOKEN` secret, you must reference it in your workflow file

Reply:
Makes sense 👍